exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -171,7 +171,7 @@ export default ExpFrameBaseComponent.extend({
     }).volatile(),
 
     allowNext: Ember.computed('cards.[]', function() {
-        if (config.validate) {
+        if (config.featureFlags.validate) {
```

Comment:
This will work for us now.

It occurs to me that the flags are used in an addon, but defined in a consuming app. This creates a situation where these frames will *only* work if the consuming app happens to specify some undocumented config variables needed by the addon.... but since these frames are very narrowly special-purpose built for ISP, this is ok for now.

We're discussing some possible changes to how experimenter would work, but until that's clarified it's hard to advise on more specific changes. For now, this is just a general comment, and something to think about if we see bugs later.